docs/articles/testing/multi-node-testing-old.md

```diff
@@ -0,0 +1,286 @@
+
+#### Configuring Individual Nodes Differently
+
+In addition to passing a `CommonConfig` object throughout all nodes in your multi-node test, you can also provide configurations for individual nodes during each test.
+
+For example: if you're taking advantage of the `akka.cluster.roles` property to have some nodes execute different workloads than others, this might be something you'd want to specify for nodes individually.
+
+The `NodeConfig` method allows you to do just that:
+
+```csharp
+NodeConfig(new List<RoleName> { First }, 
+    new List<Config> {     ConfigurationFactory.ParseString(
+        @"akka.cluster.roles =[""a"", ""c""]") });
+NodeConfig(new List<RoleName> { Second, Third }, 
+    new List<Config> { ConfigurationFactory.ParseString(
+        @"akka.cluster.roles =[""b"", ""c""]") });
+```
+
+Right after setting `CommonConfig` inside the constructor of your `MultiNodeConfig` class you can call `NodeConfig` for the specified `RoleName`s and each of them will have their `Config`s added to their `ActorSystem` configurations at startup.
+
+> [!NOTE]
+> `NodeConfig` takes precedent over `CommonConfig`
+
+#### Enabling TestTransport to Simulate Network Errors
+
+One final but important thing you might want to during the design of a multi-node test is to enable the `TestTransport`, which exposes a capability inside your tests that allows for you to create network partitions, disconnects, and latency on the fly.
+
+[!code-csharp[SurviveNetworkInstabilitySpec.cs](../../../src/core/Akka.Cluster.Tests.MultiNode/SurviveNetworkInstabilitySpec.cs?name=MultiNodeSpecConfig)]
+
+To enable the `TestTransport`, all you have to do is set `TestTransport = true` inside the `MultiNodeConfig` constructor.
+
+Once that's done, you'll be able to use the `TestConductor` inside your multi-node tests to enable all kinds of simulated network partitions.
+
+### Step 2 - Create Your MultiNodeClusterSpec or MultiNodeSpec
+
+Once you've created your `MultiNodeConfig`, you'll want to create a `MultiNodeClusterSpec` if you're using Akka.Cluster or a `MultiNodeSpec` if you just want to use Akka.Remote.
+
+We're going to show you a full code sample first and walk through how it works in detail below.
+
+[!code-csharp[RestartNode2Spec.cs](../../../src/core/Akka.Cluster.Tests.MultiNode/RestartNode2Spec.cs?name=MultiNodeSpec)]
+
+First, take note of the default public constructor:
+
+```csharp
+public RestartNode2Spec() : this(new RestartNode2SpecConfig()) { }
+```
+
+This is an XUnit restriction - there can only be one public constructor per class, and you need to pass in your `MultiNodeConfig` to the base class constructor, which is exactly what we do in the `protected` constructor.
+
+```csharp
```

Comment:
These need to be DocFx-style reference https://getakka.net/community/contributing/documentation-guidelines.html#code-samples-must-use-code-references

Ditto with the others in this file.